### DISCUSSION TITLE
Allow group by Milestones and labels

### DISCUSSION THREAD

**Discussion by brunovollmer** [community/community#6581] [Projects and Issues]
Hey everybody,

thanks for the great beta. Love it so far. One thing I'm missing so far is that you can't group by Milestones or Labels.

That would be a great addition.

Bruno

**Comment by RonnyB71** [comment]
You can group by Milestone already, but not Labels.

**Reply by brunovollmer** [reply]
Okay. The way we use it at the moment, would be definitely simplified if we could also group by labels.

**Reply by RonnyB71** [reply]
Yep. Agree.

**Comment by silopolis** [comment]
Group by label, yes please
